### PR TITLE
Fixing line comments and indentation, closes #17781

### LINKED ISSUE #17781: Comments starting with '#' are deprecated in ini files
Could you please use `//` for line comments?

https://github.com/owncloud/core/blob/master/apps/user_ldap/tests/wizard.php#L143
219 and 434

### PR DESCRIPTION
**Pull request by szepeviktor**
Please see the PR-s to 3rd-party repos in #17781.

### CODE REVIEW

**Comment by szepeviktor** [comment]
I do state that this contribution is MIT licensed.

File: apps/user_ldap/tests/wizard.php [hunk 1]
```diff
@@ -431,7 +431,7 @@ public function testCumulativeSearchOnAttributeSkipReadDN() {
 				return $b;
 			}));
 
-		# The following expectations are the real test #
+		// The following expectations are the real test
```
Comment (LukasReschke) [inline apps/user_ldap/tests/wizard.php]: The . here seems like an error?

Comment (LukasReschke) [inline apps/user_ldap/tests/wizard.php]: Ah. - I should clean my screen… My mistake :speak_no_evil:

**Comment by LukasReschke** [comment]
:+1:

**Comment by LukasReschke** [comment]
Thanks! :) - You are also more than welcome to join the party at #owncloud-dev in Freenode IRC. (though idling is recommended ;-))

**Comment by LukasReschke** [comment]
@owncloud-bot This is okay to test

**Comment by LukasReschke** [comment]
@owncloud-bot Retest this please

**Comment by szepeviktor** [comment]
@LukasReschke Please advise what to do.

**Comment by LukasReschke** [comment]
@owncloud-bot Retest this please

**Comment by LukasReschke** [comment]
@szepeviktor Was a hiccup in the CI system. No action required.

**Comment by Xenopathic** [comment]
I'm amazed that the original code worked at all. :+1:

**Pull request merged**